documentation/docs/operation/commands/history.md

```diff
@@ -81,4 +81,29 @@ Using the same example, get the starting quote value. Convert the starting ETH a
 ```
 Return % = -0.562 USDT / 2539.13 USDT
 Return % = -0.0222 %
+```
+
+
+## Net Delta
+
+Net Delta is calculated based on the difference of starting balance and current balance, while Trade Delta is tokens acquired after executing trades. Both Net Delta or Trade Delta can be positive or negative (profit or loss).
```

Comment:
Net Delta is the difference of your starting balance and current balance i.e. it takes into calculation assets spent & acquired during trades, deposits and withdrawals.

Trade Delta does not take into account deposits and withdrawals, only balance changes after executing trades in Hummingbot.

Reply:
Noted, thanks.